.gitignore

```diff
@@ -102,3 +102,5 @@ venv.bak/
 
 # mypy
 .mypy_cache/
+
+.DS_Store
```

Comment:
IMO `.DS_Store` and other OS-specific files do not belong here. You should look into adding a global `.gitignore` via:
```
# ~/.gitconfig
[core]
    excludesfile = /path/to/my/global/gitignore
```

You can find a full example [here](https://github.com/kne42/dotfiles/tree/master/git).

Reply:
got it - removing change, and replaced with global git ignore on my computer